packages/react-components/src/components/Button.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import { useRuleEngine, useFormIntl } from '@aemforms/af-react-renderer';
+import '../../dist/theme.css';
+import {State, FieldJson} from '@aemforms/af-core';
+
+const Button = (fieldjson: State<FieldJson>) => {
+   const [props, handlers] = useRuleEngine(fieldjson);
+   const i18n = useFormIntl();
+   const { label, enabled, visible } = props;
+   const formattedLabel = i18n.formatMessage({ id: `${label?.value}` });
```

Comment:
This will not work. Have to test it? 
You have to get the localization id from "props.properties['afs:translationIds']"

Reply:
Implemented!